packages/mask/src/plugins/MaskBox/hooks/useContext.ts

```diff
@@ -65,7 +67,7 @@ function useContext(initialState?: { boxId: string }) {
     const { value: paymentTokens = [] } = useFungibleTokensDetailed(
         maskBoxStatus?.payment?.map(([address]) => {
             return {
-                type: isSameAddress(address, ZERO_ADDRESS) ? EthereumTokenType.Native : EthereumTokenType.ERC20,
+                type: isNative(address) ? EthereumTokenType.Native : EthereumTokenType.ERC20,
```

Comment:
Rename it to `isNativeTokenAddress`.

Reply:
<img width="421" alt="CleanShot 2021-12-27 at 13 54 57@2x" src="https://user-images.githubusercontent.com/3842474/147439245-9c592653-9e07-4d9f-b555-ac44f54a7b31.png">